agentlocal/agentlocal.go

```diff
@@ -58,7 +60,10 @@ type Status struct {
 	ServerVersion     string   `json:"server_version"`
 
 	Agents []AgentStatus `json:"agents"`
-	// TODO latency / last ping time / connection status
+
+	ServerClockDrift time.Duration
+	ServerLatency    time.Duration
+	Connected        bool
```

Comment:
Seems to be `Connected` should be logically before other fields – they don't make sense if agent is not connected.